client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/page-list.tsx

```diff
@@ -38,12 +39,20 @@ const PageListItem = ( { label, isSelected, isDisabled }: PageListItemProps ) =>
 };
 
 interface PageListProps {
+	categories: Category[];
+	pagesMapByCategory: Record< string, Pattern[] >;
 	selectedPages: string[];
 	onSelectPage: ( selectedPage: string ) => void;
 }
 
-const PageList = ( { selectedPages, onSelectPage }: PageListProps ) => {
+const PageList = ( {
+	categories,
+	pagesMapByCategory,
+	selectedPages,
+	onSelectPage,
+}: PageListProps ) => {
 	const translate = useTranslate();
+	const categoriesInOrder = usePagesOrder( categories );
```

Comment:
[nit] The `categoriesInOrder` and `usePagesOrder` sounds mismatched.  It might be better to call them `orderedPageCategories` and `useOrderedPageCategories`.

Reply:
I think we can reuse the `useCategoriesOrder` since most of the code is the same. I've proposed this change in #83622!